WalletWasabi/Wallets/Wallet.cs

```diff
@@ -137,7 +138,18 @@ public IEnumerable<SmartTransaction> GetTransactions()
 		}
 		return walletTransactions.OrderByBlockchain().ToList();
 	}
-	
+
+	public HdPubKey CreateReceiveAddress(IEnumerable<string> destinationLabels)
```

Comment:
I am not sure about the term `Create` as technically you just have to find the next clean one. If this was in fluent project I would say it doesn't matter. But in the business code shouldn't we be more precise?
Maybe it doesn't even matter here either.

Reply:
@soosr changed to `GetNextReceiveAddress()`